### ISSUE TITLE
authconfig command is not available

### ISSUE THREAD

**Issue by msandres13** [livecd-tools/livecd-tools#179]
Since Fedora 28 authconfig is deprecated since centos8.
[(https://docs.centos.org/en-US/8-docs/advanced-install/assembly_kickstart-commands-and-options-reference/#:~:text=The%20auth%20or%20authconfig%20Kickstart,and%20package%20have%20been%20removed.&text=The%20installation%20program%20will%20automatically,to%20provide%20the%20compatibility%20layer.)]

I assume this needs to be replaced by authselect?
https://github.com/livecd-tools/livecd-tools/blob/master/imgcreate/creator.py#L791

Due to this imgcreate always shows the message authconfig command is not available.

What is the plan here? Should be authconfig be replaced with authselect?

**Comment by Conan-Kudo** [comment]
Yeah, most likely we need to change it. Does it still work with the authconfig compatibility tool or no?

**Comment by msandres13** [comment]
If you build a image which contains the authconfig compat tool the error message does not surface.
Once it is removed how it should be since authconfig is deprecated the error message surfaces.
Since the image is still created this might not be the highest priority and should eventually get fixed I guess.
